app_dart/lib/src/foundation/utils.dart

```diff
@@ -73,20 +68,6 @@ Future<List<String>> loadBranchRegExps(
   return <String>['master'];
 }
 
-// Runs a db transaction with retries.
-//
-// It uses quadratic backoff starting with 50ms and 3 max attempts.
-Future<void> runTransactionWithRetries(RetryHandler retryHandler,
```

Comment:
nit: Why was this moved out of utils.dart? I thought this was a great addition, and was hoping to import it into the CacheRequestHandler to reuse the logic there.

Reply:
I moved the function to ./services/datastore.dart without changes. I think that's a better place for it because it is checking for errors in the datastore transactions.

I have a follow up PR to move all the existing code to use the wrappers to keep this PR as short as possible.